server/src/main/java/org/elasticsearch/index/search/MatchQuery.java

```diff
@@ -175,6 +231,8 @@ public void writeTo(StreamOutput out) throws IOException {
 
     protected boolean autoGenerateSynonymsPhraseQuery = true;
 
+    protected SynonymQueryStyle synonymQueryStyle = SynonymQueryStyle.BLENDED_TERMS;
```

Comment:
Why not use `DEFAULT_SYNONYM_QUERY_STYLE` as a value ? This way if the default is ever changed then one has to change it at just one place.